<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YouTube Downloader</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon/icons8-youtube-squared-48.png</normaloff>icon/icons8-youtube-squared-48.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title_label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>521</width>
      <height>57</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
color:black;</string>
    </property>
    <property name="text">
     <string> YouTube Downloader </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="url_lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>511</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
color:black;</string>
    </property>
   </widget>
   <widget class="QLabel" name="videourl_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>244</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
color:black;
</string>
    </property>
    <property name="text">
     <string>Enter video URL:</string>
    </property>
   </widget>
   <widget class="QLabel" name="format_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>206</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
color:black;

</string>
    </property>
    <property name="text">
     <string>Video Format:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="format_groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QRadioButton" name="mp3_radioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>78</width>
       <height>37</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:black;</string>
     </property>
     <property name="text">
      <string>Mp3</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="mp4_radioButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>78</width>
       <height>37</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:black;</string>
     </property>
     <property name="text">
      <string>Mp4</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="video_playlist_label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>207</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
color:black;

</string>
    </property>
    <property name="text">
     <string>Video/Playlist:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="video_groupBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>230</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="video_radioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>93</width>
       <height>37</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:black;
</string>
     </property>
     <property name="text">
      <string>Video</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="playlist_radioButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>108</width>
       <height>37</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:black;</string>
     </property>
     <property name="text">
      <string>Playlist</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="resolution_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>163</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
color:black;

</string>
    </property>
    <property name="text">
     <string>Resolution:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="resolution_comboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="directory_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>220</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
color:black;

</string>
    </property>
    <property name="text">
     <string>Your Directory:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="directory_lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>631</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
color:black;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browse_pushButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>450</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#FF0000;
color:#F5F8FA;</string>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>130</y>
      <width>231</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
background-color:#F5F8FA;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submit_pushButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>80</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#FF0000;
color:#F5F8FA;</string>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="check_pushButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>380</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#FF0000;
color:#F5F8FA;
</string>
    </property>
    <property name="text">
     <string>Check</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>81</width>
      <height>57</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
color:black;
	  
	
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>icon/1298778_youtube_play_video_icon.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>831</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
